diff --git a/node_modules/iztro/lib/utils/index.js b/node_modules/iztro/lib/utils/index.js
index 90e1dc4..1f2e2ae 100644
--- a/node_modules/iztro/lib/utils/index.js
+++ b/node_modules/iztro/lib/utils/index.js
@@ -94,9 +94,7 @@ exports.fixEarthlyBranchIndex = fixEarthlyBranchIndex;
  * 调整农历月份的索引
  *
  * 正月建寅（正月地支为寅），fixLeap为是否调整闰月情况
- * 若调整闰月，则闰月的前15天按上月算，后面天数按下月算
- * 比如 闰二月 时，fixLeap 为 true 时 闰二月十五(含)前
- * 的月份按二月算，之后的按三月算
+ * 若调整闰月，则整个闰月按下月算（閏月以下月論）
  *
  * @param {string} solarDateStr 阳历日期
  * @param {number} timeIndex 时辰序号
@@ -107,7 +105,7 @@ var fixLunarMonthIndex = function (solarDateStr, timeIndex, fixLeap) {
     var _a = (0, lunar_lite_1.solar2lunar)(solarDateStr), lunarMonth = _a.lunarMonth, lunarDay = _a.lunarDay, isLeap = _a.isLeap;
     // 紫微斗数以`寅`宫为第一个宫位
     var firstIndex = data_1.EARTHLY_BRANCHES.indexOf('yinEarthly');
-    var needToAdd = isLeap && fixLeap && lunarDay > 15 && timeIndex !== 12;
+    var needToAdd = isLeap && fixLeap && timeIndex !== 12;
     return (0, exports.fixIndex)(lunarMonth + 1 - firstIndex + (needToAdd ? 1 : 0));
 };
 exports.fixLunarMonthIndex = fixLunarMonthIndex;
